pallets/creditcoin/src/weights.rs

```diff
@@ -464,4 +464,16 @@ impl<T: frame_system::Config> crate::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+
+	fn set_burn_gate_contract() -> Weight {
+		Weight::from_parts(0,0)
+	}
+
+	fn set_burn_gate_faucet_address() -> Weight {
+		Weight::from_parts(0,0)
+	}
+
+	fn request_collect_coins_v2() -> Weight {
+		Weight::from_parts(0,0)
+	}
```

Comment:
@nathanwhit do these weights look correct ?

Reply:
I would expect at a minimum that the `collect_coins_v2` would be similar to the original `collect_coins`. These definitely need to be updated.